maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlAssertVisual.kt

```diff
@@ -0,0 +1,24 @@
+package maestro.orchestra.yaml
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import java.lang.UnsupportedOperationException
+
+private const val DEFAULT_DIFF_THRESHOLD = 95
```

Comment:
Should this be exposed as an environment variable (assuming you don't want to go as far as to expose it to workspace config at this point)?